pkgs/tools/games/ukmm/default.nix

```diff
@@ -6,27 +6,28 @@
 , wrapGAppsHook3
 , libglvnd
 , libxkbcommon
+, openssl
 , nix-update-script
 }:
 
 rustPlatform.buildRustPackage rec {
   pname = "ukmm";
-  version = "0.12.1";
+  version = "0.13.0";
 
   src = fetchFromGitHub {
     owner = "NiceneNerd";
     repo = pname;
```

Comment:
```suggestion
    repo = "ukmm";
```

Reply:
Oh yep thanks! I forgot this was still using `pname` here.